src/tests/issues.targets

```diff
@@ -1245,6 +1245,25 @@
         <ExcludeList Include="$(XunitTestBinBase)/JIT\Directed\debugging\poisoning\poison\*" />
     </ItemGroup>
 
+    <!-- The mobile wrappers don't support multiple entry points. The tests will be enabled once the parallel AOT compilation on Helix is introduced in https://github.com/dotnet/runtime/pull/87773. -->
```

Comment:
Why do we have both an ExcludeList entry and CLRTestTargetUnsupported for many of these? Someone might think deleting an entry from here will make us run the test, but it won't, because it's double disabled.

Reply:
You are absolutely right, the test exclusion works locally but fails on the CI, so I decided to disable them temporarily by using `CLRTestTargetUnsupported` to test the flow. Since that worked, I have to fix the test exclusion logic to work on the CI.

Reply:
Updated, only tests without entry points are disabled with CLRTestTargetUnsupported.

Reply:
Sorry I did not understand in the end, why can't we unify what needs to be excluded?

For example 

- why can't we add here (and remove setting of `CLRTestTargetUnsupported` in the project file):
    ```xml
    <ExcludeList Include="$(XunitTestBinBase)/nativeaot/SmokeTests/SharedLibrary/">
        <Issue>needs triage</Issue>
    </ExcludeList>
    ```

- or the other way around specify `CLRTestTargetUnsupported` for all of these test project files, and remove them from `ExcludeList`?

Reply:
There are two ways to disable the tests:
 - Adding the CLRTestTargetUnsupported property, which prevents test AOT compilation
 - Adding an ExcludeList item to filter out tests after the build and compilation process

The SharedLibrary fails to compile due to `<NativeLib>Shared</NativeLib>` resulting in `no entrypoint module` failure.

The current approach generates Xunit wrappers per test directory, performs AOT compilation on build machines, and creates a bundle per test directory. Some tests have multiple "entry points" per test directory. Even though tests are excluded using the `ExcludeList`, the wrappers are generated before the exclusion, and the payloads are sent to Helix.

General idea is to implement a unified pipeline for iOS tests using AppleTestRunner introduced in https://github.com/dotnet/runtime/pull/34976. This approach should allow for multiple entry points per test directory and enable AOT compilation on Helix.

Since we plan avoid XUnit wrappers in future, I suggest we disable them using the CLRTestTargetUnsupported until they are updated to use the AppleTestRunner, and avoid any further workarounds using the wrappers and the ExcludeList items.